pyigm/clustering/xcorr_utils.py

```diff
@@ -322,7 +386,175 @@ def spline_sensitivity(galreal, Nmin=20, magmin=17., magmax=26., delta_mag=0.5,
     return magbins, VALS, SPL
 
 
-def random_gal(galreal, Nrand, magbins, SPL):
+def random_abs_zmnx(absreal, Nrand, zmnx, wrest, dv_Galactic=100.):
```

Comment:
This line marks how far I got reviewing this for tonight. Will continue tomorrow, but will submit previous comments so you can address them in the meantime if you wish.

Reply:
Thanks!